make/run-cicd-script.mk

```diff
@@ -1,5 +1,5 @@
 
-OWNER_AND_BRANCH_LOCATION=codeready-toolchain/toolchain-cicd/master
+OWNER_AND_BRANCH_LOCATION=MatousJobanek/toolchain-cicd/support-deploy-latest
```

Comment:
I guess this needs to be reverted?

Reply:
yes, sure - this is to make the e2e tests pass and also to verify the change :-)

Reply:
updated a05a451